src/generators/typescript/renderers/EnumRenderer.ts

```diff
@@ -85,8 +85,8 @@ ${this.indent(this.renderBlock(content, 2))}
 }
 
 export const TS_DEFAULT_ENUM_PRESET: EnumPreset<EnumRenderer> = {
-  async self({ renderer }) {
-    return `export ${await renderer.defaultSelf()}`;
+  self({ renderer }): Promise<string> {
```

Comment:
You don't need to define returned type, it's redundant.

Reply:
Removed 🙂